client/app/pods/components/question-multi-radio/component.js

```diff
@@ -0,0 +1,9 @@
+import QuestionComponent from 'tahi/pods/components/question/component';
```

Comment:
Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.<br>'QuestionComponent' is not defined.<br>'from' is not defined.